### ISSUE TITLE
SfB-Online: token is not added to the request header when getting photo

### ISSUE THREAD

**Issue by ybutkevych** [OfficeDev/skype-web-sdk-samples#2]
Hi guys,

On the 'contacts' page I cannot see the photo of the contact. The problem is that the authorization token is not being added to the request header.
![sfb-online-photo](https://cloud.githubusercontent.com/assets/10098526/14145989/b0821cca-f69e-11e5-8d94-44a76d9c8cb7.png)

Thanks

**Comment by antonkh** [comment]
These GET requests use the `Cookie: cwt_ucwa=...` header for authorization. I suppose the cookie is sent, but for some reason the server rejected it (perhaps the cookie expired?). Did you have a chance to capture the response headers? They may explain this 401.

**Comment by MJZawacki** [comment]
I'm getting the same error. This specific request doesn't even have the cookie in the request. I was trying to also look for specific requests in the network log that originally set that cookie and can't find it.

**Comment by MikeDev2016** [comment]
I am also receiving this same error.

**Comment by sidharthv** [comment]
Yes, we are aware of this issue, and working on a fix. The server does not provide the Access-Control-Allow-Credentials header when returning the response to POST /applications, causing the browser to lose the cookie. This requires server changes and hence will take some time to roll out.

**Comment by praneethw** [comment]
The photo seems to be loading in IE 11 as the **cwt_ucwa** cookie is being passed as expected.
However, Chrome and Firefox does not send this cookie, hence causing a 401 error.
Would you have any idea if this is a known issue and time of fix?
